src/spryker/application/overlay/phpstan.neon

```diff
@@ -0,0 +1,21 @@
+includes:
+    - vendor/spryker-sdk/phpstan-spryker/extension.neon
+
+parameters:
+    excludes_analyse:
+        - %rootDir%/../../../src/Generated/*
+        - %rootDir%/../../../src/Orm/*
+        # ignoring following dir temporarily due to errors in core
+        - %rootDir%/../../../src/Pyz/Zed/DataImport/Business/Model/*
```

Comment:
only added this line in addition to core spryker file. I don't know why the phpstan tests are failing for the files in this directory, I have spent quite some time to debug and also asked in Spryker community slack about it, until then I have excluded this dir to not block this PR